ts/src/bitso.ts

```diff
@@ -1695,6 +1695,10 @@ export default class bitso extends Exchange {
         return this.safeString (statuses, status, status);
     }
 
+    nonce () {
```

Comment:
@mironsv next time, please update the typescript file instead, all the other languages (js, py, PHP) are automatically transpiled

Reply:
ok, I got your point!
thank you @carlosmiei